frontend/pages/policies/ManagePoliciesPage/components/AddPolicyModal/AddPolicyModal.tsx

```diff
@@ -91,23 +135,43 @@ const AddPolicyModal = ({
     );
   });
 
+  const renderNoResults = (): JSX.Element => {
+    return (
+      <>
+        There are no results that match your filters.{" "}
+        <CustomLink
+          text="Everyone can contribute"
+          url={CONTRIBUTE_TO_POLICIES}
+          newTab
+        />
+      </>
+    );
+  };
   return (
     <Modal
       title="Add a policy"
       onExit={onCancel}
       className={baseClass}
-      width="xlarge"
+      width="large"
     >
       <>
-        <div className={`${baseClass}__create-policy`}>
+        <div className={`${baseClass}__description`}>
           Choose a policy template to get started or{" "}
           <Button variant="text-link" onClick={onCreateYourOwnPolicyClick}>
             create your own policy
           </Button>
           .
         </div>
+        <Dropdown
+          value={platform}
+          className={`${baseClass}__platform-dropdown`}
+          options={PLATFORM_FILTER_OPTIONS}
+          searchable={false}
+          onChange={onPlatformFilterChange}
+          tableFilterDropdown
+        />
         <div className={`${baseClass}__policy-selection`}>
-          {policiesAvailable}
+          {filteredPolicies.length > 0 ? policiesAvailable : renderNoResults()}
```

Comment:
since `filteredPolicies` and `policiesAvailable` (see suggested rename) reflect the same state, it'd be slightly clearer to me to use the same one for the whole ternary, i.e.:
`policiesAvailable.length > 0 ? policiesAvailable : renderNoResults()`

Reply:
I renamed both `filteredPolicies.length` and `policiesAvailable` to use `renderFilteredPolicies.length` and `renderFilteredPolicies` respectively.

Thanks for helping keep our code 🧼